libc-top-half/musl/src/thread/pthread_getattr_np.c

```diff
@@ -20,5 +23,9 @@ int pthread_getattr_np(pthread_t t, pthread_attr_t *a)
 			l += PAGE_SIZE;
 		a->_a_stacksize = l;
 	}
+#else
```

Comment:
Perhaps we can leave the `if / else` above and just replace the else block. 

It looks like that `if` block covers secondary threads and the `else` block covers the main thread?  We could use the `__stack_high` / `__stack_low` globals for the else case I think?

Reply:
The idea was to fill these fields in main thread struct and handle everything the same way. Otherwise this would need to duplicate the logic for getting stack bounds, fallback to __heap_base, etc...

Reply:
Replaced only `else` block.